sdk/webpubsub/Azure.Messaging.WebPubSub/tests/WebPubSubParseConnectionStringTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+
+using NUnit.Framework;
+
+namespace Azure.Messaging.WebPubSub.Tests
+{
+    [TestFixture]
+    public class WebPubSubParseConnectionStringTests
+    {
+        private static readonly JwtSecurityTokenHandler JwtTokenHandler = new JwtSecurityTokenHandler();
+
+        [TestCase("Endpoint=https://host;AccessKey=abcdefg;Version=1.0;", "https://host", "abcdefg")]
```

Comment:
I hope these don't trigger credscan?  You might need to make it a format string and merge in the access key in code.